File: class18/cpp/AP1810-TabuadaEnquanto.cpp
#include <iostream>
#include <iomanip>
int main() {
    int n;
    std::cout << "Digite um número: "; std::cin >> n;
    int i = 1;
    while (i <= 10) {
        std::cout << n << " × " << std::setw(2) << i << " = " << std::setw(3) << n*i << std::endl;
        i++;
    }
    return 0;
}

File: class18/java/AP1810-TabuadaEnquanto.java
import java.util.Scanner;
public class TabuadaEnquanto {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.print("Digite um número: ");
        int n = sc.nextInt(), i = 1;
        while (i <= 10) {
            System.out.printf("%d × %2d = %3d%n", n, i, n*i);
            i++;
        }
        sc.close();
    }
}